Kernel/Memory/MemoryManager.cpp

```diff
@@ -291,7 +291,11 @@ UNMAP_AFTER_INIT void MemoryManager::parse_memory_map()
                 global_data.physical_memory_ranges.append(PhysicalMemoryRange { PhysicalMemoryRangeType::Usable, start_address, length });
                 break;
             case (MULTIBOOT_MEMORY_RESERVED):
-                global_data.physical_memory_ranges.append(PhysicalMemoryRange { PhysicalMemoryRangeType::Reserved, start_address, length });
+#if ARCH(I386) || ARCH(X86_64)
+                // Workaround for QEMU commit 8504f12.
```

Comment:
Please:
- Add a link to said commit
- Go into slightly more detail so others can understand what this is doing

Comments are free :)

Reply:
:100: